### PR TITLE
[Merged by Bors] - docs: prevent lists running onto a single line

### PR DESCRIPTION
**Pull request by eric-wieser**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/LocalHomeomorph.lean [hunk 1]
```diff
@@ -22,14 +22,14 @@ instead of `e.toFun x` and `e.invFun x`.
...
 ## Main definitions
 
-`Homeomorph.toLocalHomeomorph` : associating a local homeomorphism to a homeomorphism, with
-                                 `source = target = Set.univ`;
-`LocalHomeomorph.symm` : the inverse of a local homeomorphism
-`LocalHomeomorph.trans` : the composition of two local homeomorphisms
-`LocalHomeomorph.refl` : the identity local homeomorphism
-`LocalHomeomorph.ofSet` : the identity on a set `s`
-`LocalHomeomorph.EqOnSource` : equivalence relation describing the "right" notion of equality
-                               for local homeomorphisms
```
Comment (eric-wieser) [inline Mathlib/Topology/LocalHomeomorph.lean:32]: This rendered as

image

**Review by grunweg — APPROVED** [review]

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Comment by eric-wieser** [comment]
bors merge

**Comment by eric-wieser** [comment]
bors merge

**Pull request closed without merging**